examples/pixelbased_decoding.py

```diff
@@ -0,0 +1,95 @@
+"""
+.. _tutorial_pixel_based_decoding:
+
+Pixel-Based Decoding
+====================
+
+Pixel-based decoding is the approach of localizing and :term:`decoding <Decoding>` molecules
+(e.g. RNA transcripts or :term:`rolonies <Rolony>`) that does not rely on algorithms to find
+spots by fitting Gaussian profiles or local intensity maxima. Instead of finding spots to be
+decoded, it decodes every pixel and then connects potential pixels with the same
+:term:`codeword <Codeword>` into spots. The strength of this approach is it works on dense data
+and noisy data where spot finding algorithms have a hard time accurately detecting spots. The
+weakness is that it is prone to false positives by decoding noise that would normally be ignored
+by spot finding algorithms. That is why pixel-based decoding is better suited for error-robust
+assays such as MERFISH where the :term:`codebook <Codebook>` is designed to filter out noise.
+
+.. image:: /_static/design/decoding_flowchart.png
+   :scale: 50 %
+   :alt: Decoding Flowchart
+   :align: center
+
+Unlike :ref:`tutorial_spot_based_decoding` that has a variety of classes and methods to customize a
+workflow, pixel-based decoding is straight-forward and only requires one class in starfish:
+:py:class:`.PixelSpotDecoder`. The only decoding algorithm option is the same as
+:py:class:`.MetricDistance` but applied to each pixel. That means normalizing images is very
+important for accurate decoding and it needs vector magnitude and distance thresholds. See
+:ref:`howto_metricdistance` for more details.
+
+After decoding pixels, :py:class:`.PixelSpotDecoder` combines connected  pixels into spots with
+:py:func:`skimage.measure.label` and :py:func:`skimage.measure.regionprops`, which returns spot
+locations and attributes. Spots that don't meet size thresholds ``min_area`` and ``max_area`` are
+marked as not passing thresholds in the :py:class:`.DecodedIntensityTable`. In addition, the results
+from connected component analysis that do not fit in the :py:class:`.DecodedIntensityTable` (e.g.
+spot attributes and labeled images) are returned in a :py:class:`.ConnectedComponentDecodingResult`.
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from copy import deepcopy
+from starfish import data, FieldOfView, display
+from starfish.image import Filter
+from starfish.spots import DetectPixels
+from starfish.types import Axes, Features, Levels
+
+# Load MERFISH data
+experiment = data.MERFISH(use_test_data=True)
+imgs = experiment.fov().get_image(FieldOfView.PRIMARY_IMAGES)
+
+# filter and deconvolve data
+ghp = Filter.GaussianHighPass(sigma=3)
+dpsf = Filter.DeconvolvePSF(num_iter=15, sigma=2, level_method=Levels.SCALE_SATURATED_BY_CHUNK)
+glp = Filter.GaussianLowPass(sigma=1)
+ghp.run(imgs, in_place=True)
+dpsf.run(imgs, in_place=True)
+glp.run(imgs, in_place=True)
+
+# scale data with user-defined factors to normalize images
```

Comment:
```suggestion
# scale data with user-defined factors to normalize images.  For this data set, the scale factors are stored in experiment.json.
```

^^ suggested this because it's not required that the scale factors be in experiment.json.

Reply:
👍